src/styles/todoapp.scss

```diff
@@ -129,5 +129,9 @@
     &:active {
       text-decoration: none;
     }
+
+    &:disabled {
```

Comment:
Please ensure that the class names follow the naming convention of using double underscores for nested elements and double dashes for modifiers. If this is a modifier, use '--disabled'.